docs/3-managed-service.md

```diff
@@ -74,88 +65,62 @@ Once your Display Name matches the format above, we'll promptly verify your acco
 
 ### Message Encryption
 
-By default, Hush Line doesn't require a PGP key, but it's strongly encouraged. After adding your PGP key, only you can read the messages you receive.
-
-As a baseline security measure, we encrypt your message content by default on our server. While this protects the contents of our database, if the encryption key becomes compromised, it can be decrypted. But if you add your PGP key, any message received will only be able to be decrypted by you. If you expect to receive messages containing sensitive content, enabling this feature is strongly encouraged.
-
-<img src="img/inbox.encrypted.png">
+By default, Hush Line requires a PGP key to receive messages, making your messages only readable by you.
 
-### Mailvelope
-
-After installing the Mailvelope extension in your browser, you can decrypt your messages directly in the app. This is highly recommended, as it's one of the easiest and most straightforward methods of decrypting your messages.
+## Settings
 
-<img src="img/inbox.mailvelope.png">
+### Profile Settings
 
-You'll see a new interface where the encrypted message was. Click "Show message" to decrypt. You'll enter the password you created when setting up your PGP key, and the contents will be visible.
+![Settings - Directory](https://github.com/user-attachments/assets/54f18599-0f7f-41b7-ad6a-81166bd6cc6a)
 
-<img src="img/inbox.decrypted.png">
+#### Display Name
 
-## Settings
+Users can set a human-readable display name so that someone submitting a message can see "Submit a message to Science & Design" rather than "Submit a message to scidsg". As a security measure, if a user changes their display name after verification, they'll lose that status and must re-verify their account.
 
-### Profile Settings
+#### Public Directory
 
-Users can tailor Hush Line to their needs, from adding a display name to adding multiple usernames.
+Users may opt in to being listed on our public directory, making it easy for anyone to find their Hush Line account.
 
-<img src="img/settings.profile.png">
+#### Verified Addresses
 
-#### Display Name
+Verified addresses establishes a relationship between your Hush Line account and another website. Verification is automatic, and all you need to do is include a link to your Hush Line profile containing `rel="me"` on the page of the address you want verified.
 
-Users can set a human-readable display name so that someone submitting a message can see "Submit a message to Science & Design" rather than "Submit a message to scidsg".
+For example, if I want to verify my personal website `glennsorrentino.com` I would add this to my page:
 
-##### Verification
+```html
+<a href="https://tips.hushline.app/to/glenn" rel="me">Hush Line</a>
+```
 
-As a security measure, if a user changes their display name after verification, they'll lose that status and must re-verify their account.
+In your Hush Line settings, when I add `https://glennsorrentino.com` you'll see a verified checkmark.
```

Comment:
I wonder if this is worth a screenshot as users might try adding urls to their bio instead of "Extra Fields".

Reply:
Maybe add a note that says "Only urls using https protocol are eligible for verification"?

Reply:
Good call. I also updated the criteria for verification. It occurs to me that we can simply use verified URLs as a proxy for authenticity of the receiver. Both inbound to Hush Line, or outbound to Mastodon.